tests/opentelemetry/test_context.py

```diff
@@ -129,3 +133,48 @@ async def coro(i):
         await coro(2)
         await coro(3)
         await coro(4)
+
+
```

Comment:
Can we make the test names a bit more descriptive. From a quick glance it's hard to tell which parts of the otel/dd baggage api is being tested. This makes these tests a bit challenging to maintain.